src/style.css

```
@@ -46,11 +48,11 @@ body {
   color: #00acdc;
 }
 
-li:first-child .nav-link {
+.nav-list li:first-child {
```

```suggestion
.nav-item:first-child {
```
використовуй, будь ласка, класи для стилів по максимуму